tracee-ebpf/tracee/net_events.go

```diff
@@ -143,23 +190,23 @@ func (t *Tracee) processNetEvents() {
 				switch netEventId {
 				case DebugNetSecurityBind:
 					fmt.Printf("%v  %-16s  %-7d  debug_net/security_socket_bind LocalIP: %v, LocalPort: %d, Protocol: %d\n",
```

Comment:
All these printings print the same thing.
I think you should put them inside a const and maybe even create a print function for this message.

Reply:
agreed , but i wondering if this is major change for one PR or its fine.
what you think ?

Reply:
also i think that all of that net events that arent NET_PACKET event will eventually need to move to other place and get a proper handling (also think about the IRC,DNS, HTTP,etc events that coming) i think its better to create a different PR for that change dont you ?

Reply:
> agreed , but i wondering if this is major change for one PR or its fine. what you think ?

I don't think that putting it in this PR makes it too large. This is the required amount of effort for the size of this PR.



> also i think that all of that net events that arent NET_PACKET event will eventually need to move to other place and get a proper handling (also think about the IRC,DNS, HTTP,etc events that coming) i think its better to create a different PR for that change dont you ?

I think that when it comes, someone will create sub-function for it. But handling the debug printing for now should be done properly.